modules/sdk/src/main/scala/org/tessellation/sdk/snapshot/package.scala

```diff
@@ -0,0 +1,18 @@
+package org.tessellation.sdk
+
+import org.tessellation.schema.snapshot.Snapshot
+import org.tessellation.schema.{Block, SnapshotOrdinal}
+import org.tessellation.sdk.infrastructure.consensus.Consensus
+import org.tessellation.security.signature.Signed
+
+package object snapshot {
+
+  type SnapshotEvent = Signed[Block[_]]
```

Comment:
You don't need this type to be parameterized by `B <: Block[_]` in order to not operate on generic `Block`?